### DISCUSSION TITLE
When I press REPLY - why no text of email included?

### DISCUSSION THREAD

**Discussion by arabesso** [bkaankose/Wino-Mail#254] [General]
Hello!

In mail and calendar (also in thebat, thunderbird) if you press reply button - it creates new email with text of previous communication in bottom. 
When I press reply in wino mail - it like "create new email", no previous message text included.

**Comment by bkaankose** [comment]
Hello, this effort is being tracked on this issue #227

**Reply by arabesso** [reply]
> Hello, this effort is being tracked on this issue #227

thanks for info! 
Sorry, looks like I am not verified everything. It nice to know what wino mail development/fixes is active!
